File: eTuristickaAgencija.Models/Hotel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTuristickaAgencija.Models
{
    public class Hotel
    {
        public int Id { get; set; }
        public byte[] Slika { get; set; }
        public string Naziv { get; set; }
        public int GradId { get; set; }
        public int BrojZvjezdica { get; set; }
        public virtual Grad Grad { get; set; }
    }
}

File: UI/eturistickaagencija_mobile/lib/model/hotel.dart
import 'package:json_annotation/json_annotation.dart';

part 'hotel.g.dart';

@JsonSerializable()
class Hotel{
int? id;
String? naziv;
int? brojZvjezdica;
String? slika;
int? gradId;

Hotel(this.id,this.naziv,this.brojZvjezdica,this.slika,this.gradId);

factory Hotel.fromJson(Map<String,dynamic>json)=>_$HotelFromJson(json);
Map<String,dynamic>toJson()=>_$HotelToJson(this);
}
